Readme.md

```diff
@@ -395,6 +395,16 @@ cp config.ini-sample ~/.elodie/config.ini
 
 If you're an english speaker then you will probably want to add `prefer_english_names=True` to the `[MapQuest]` section else you'll have cities named using the local language.
 
+## Developing
+
+Install requirements similar to installing as a user and then run tests. NOTE different requirement file(s):
+
+```
+python -m pip install -r elodie/tests/requirements.txt
```

Comment:
An improvement here would probably be to reference dependent requirements from here and just install from a single `requirements.txt` file.

```
-r ../../requirements.txt
-r ../plugins/googlephotos/requirements.txt
```
```

Reply:
Actually, looks like the first line is already in there. Just need the latter.

Then this line can also be deleted.
https://github.com/jmathai/elodie/blob/master/.circleci/config.yml#L15